Source/control.cpp

```diff
@@ -1719,9 +1722,35 @@ static int DrawDurIcon4Item(CelOutputBuffer out, ItemStruct *pItem, int x, int c
 			break;
 		}
 	}
-	if (pItem->_iDurability > 2)
-		c += 8;
-	CelDrawTo(out, x, -17 + PANEL_Y, pDurIcons, c, 32);
+
+	int y = -17 + PANEL_Y;
+
+	//Calculate how much of the icon should be gold and red
+	int height = 32;

```

Comment:
Can we avoid this magic number? Isn't there a constant somewhere we can use? If not, can we define it?

Reply:
This is basically that.